scripts/server_scripts/mock_health_event_publisher.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+# pylint: disable=redefined-outer-name, import-error, broad-except, unspecified-encoding
+# pylint: disable=protected-access, inconsistent-return-statements
```

Comment:
@seagate-sarang-sawant I have fixed some codacy issues which didn't affect script behavior. But fixing some issues can cause change in script behavior so I have added pylint rules to disable some of the warnings.
Please let me know if this looks good to you.

Reply:
Applying Codacy skips at module level will skip them for all the Classes and functions. Ideally if you anyway want to skip pylint static analysis issue, you can limit the `pylint: disable` to that method or class.

Reply:
@seagate-sarang-sawant Modified code to apply codacy rules at method level. Please check.